lib/util.js

```diff
@@ -128,6 +128,30 @@ exports.chooseFormat = (formats, options) => {
       format = formats[formats.length - 1];
       break;
 
+    case 'highestaudio':
+      formats = exports.filterFormats(formats, 'audio');
+      format = formats[0];
```

Comment:
Need to use `let`, otherwise, without a keyword, it's declared as a global

Reply:
format is declared above in https://github.com/TimeForANinja/node-ytdl-core/blob/master/lib/util.js#L120 to return the choosen format and formats is declared as param passed to the func
i would actually just overwrite those